sbt-plugin/src/main/scala/kalix/sbt/KalixPlugin.scala

```diff
@@ -52,88 +59,99 @@ object KalixPlugin extends AutoPlugin {
   val KalixSdkVersion = BuildInfo.version
   val KalixProtocolVersion = BuildInfo.protocolVersion
 
-  override def projectSettings: Seq[sbt.Setting[_]] = Seq(
-    libraryDependencies ++= Seq(
-      "io.kalix" % "kalix-sdk-protocol" % KalixProtocolVersion % "protobuf-src",
-      "com.google.protobuf" % "protobuf-java" % "3.17.3" % "protobuf",
-      "io.kalix" %% "kalix-scala-sdk-protobuf-testkit" % KalixSdkVersion % Test),
-    Compile / PB.targets +=
-      gen(
-        (akkaGrpcCodeGeneratorSettings.value :+ KalixGenerator.enableDebug)
-        ++ rootPackage.value.map(KalixGenerator.rootPackage)) -> (Compile / sourceManaged).value,
-    Compile / temporaryUnmanagedDirectory := (Compile / crossTarget).value / "kalix-unmanaged",
-    Test / temporaryUnmanagedDirectory := (Test / crossTarget).value / "kalix-unmanaged-test",
-    protobufDescriptorSetOut := (Compile / resourceManaged).value / "protobuf" / "descriptor-sets" / "user-function.desc",
-    rootPackage := None,
-    // FIXME there is a name clash between the Akka gRPC server-side service 'handler'
-    // and the Kalix 'handler'. For now working around it by only generating
-    // the client, but we should probably resolve this before the first public release.
-    Compile / akkaGrpcGeneratedSources := Seq(AkkaGrpc.Client),
-    Compile / PB.targets ++= Seq(
-      genUnmanaged((akkaGrpcCodeGeneratorSettings.value :+ KalixGenerator.enableDebug)
-      ++ rootPackage.value.map(KalixGenerator.rootPackage)) -> (Compile / temporaryUnmanagedDirectory).value),
-    Compile / PB.generate := (Compile / PB.generate)
-      .dependsOn(Def.task {
-        protobufDescriptorSetOut.value.getParentFile.mkdirs()
+  override def projectSettings = {
+    addCommandAlias("runAll", "runDockerCompose;run") ++
+    Seq(
+      dockerComposeFile := "docker-compose.yml",
+      logConfig := "logback-dev-mode.xml",
+      userFunctionPort := 8080,
+      runDockerCompose := dockerComposeUp(dockerComposeFile.value),
+      libraryDependencies ++= Seq(
+        "io.kalix" % "kalix-sdk-protocol" % KalixProtocolVersion % "protobuf-src",
+        "com.google.protobuf" % "protobuf-java" % "3.17.3" % "protobuf",
+        "io.kalix" %% "kalix-scala-sdk-protobuf-testkit" % KalixSdkVersion % Test),
+      run / javaOptions ++= Seq(
+        "-Dkalix.user-function-interface=0.0.0.0",
+        s"-Dlogback.configurationFile=${logConfig.value}",
+        s"-Dkalix.user-function-port=${userFunctionPort.value}"),
```

Comment:
This is also a change. Previously we were passing it directly in the build.sbt file. Not needed and can we use sbt settings instead.